src/utils/HTTPTransport.ts

```diff
@@ -0,0 +1,95 @@
+type Options = {
+  method?: string,
```

Comment:
Можно определить const METHODS как enum Method (enum обычно именуется в формате PascalCase) и добавить тип
```suggestion
  method?: Method,
```